Use of undeclared module `bson`

Hi, thank you for maintaining this fork!

I can't seem to compile this crate with the `sqlx-sqlite` feature:
```rust
% cargo check
    Checking rocket_auth2 v0.6.1
error[E0433]: failed to resolve: use of undeclared crate or module `bson`
   --> /Users/centdemeern1/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rocket_auth2-0.6.1/src/db/sqlite/mod.rs:139:21
    |
139 |         let bytes = bson::to_vec(self)?;
    |                     ^^^^ use of undeclared crate or module `bson`

error[E0433]: failed to resolve: use of undeclared crate or module `bson`
   --> /Users/centdemeern1/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/rocket_auth2-0.6.1/src/db/sqlite/mod.rs:149:12
    |
149 |         Ok(bson::from_slice(bytes)?)
    |            ^^^^ use of undeclared crate or module `bson`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `rocket_auth2` (lib) due to 2 previous errors
```

`bson` only seems to be included with the `sled` feature. Is this a mistake?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,7 +21,7 @@ all-features = true
 sqlx-sqlite = ["sqlx/sqlite"]
 sqlx-postgres = ["sqlx/postgres"]
 sqlx-mysql = ["sqlx/mysql"]
-sled = ["dep:sled", "dep:bson"]
+sled = ["dep:sled"]
 
 
 [dependencies]
@@ -36,7 +36,7 @@ fehler = ">=1.0.0"
 chrono = ">=0.4.39"
 validator = { version = ">=0.20.0", features = ["derive", "unic"] }
 futures = ">=0.3.31"
-bson = { version = "2.13", optional = true }
+bson = { version = "2.13" }
 sled = { version = ">=0.34", optional = true }
 
 

